### PR TITLE
Change AE2 storage component recipes

### PR DESCRIPTION
**Pull request by Prometheus0000**
Makes them not recursive. See #7839.

### CODE REVIEW

**Comment by Dream-Master** [comment]
They ass cheap now. Replace a component with a plate ?

**Comment by Glease** [comment]
@Dream-Master they need to be cheaper than drawers. they are more tps friendly and is more pleasing to look at than a huge solid drawer cube. 

currently the only reason to use drawers at that scale is because the alternative is too expensive.

**Comment by Dream-Master** [comment]
And if you change it please add a control circuit to the recipe

**Comment by Prometheus0000** [comment]
See the discussion on the issue and in #github-discussion. And the previous discussion and PR.

**Comment by Prometheus0000** [comment]
I knew I forgot something.

**Comment by Dream-Master** [comment]
yeah but plates phew? Maybe we can think of a better recipe component for a storage then a plate ?

**Comment by Prometheus0000** [comment]
Wouldn't the data be stored on the processor+chips? With the plates keeping it together and the crystal as some kind of medium?

**Comment by Dream-Master** [comment]
compared to the old recipe we set recipe from 100% Material cost to 1%

**Comment by Prometheus0000** [comment]
Is that bad? People weren't using them before.

**Comment by Dream-Master** [comment]
Even vanilla recipe more expensive
https://ftbwiki.org/4k_ME_Storage_Component

**Comment by GTNH-Afx237v7** [comment]
"Set recipe costs from 100% to 1%" ->Yes because the old recipe was insane. Compared to super/quantum chests the AE components were just ridicuoulusly expensive; and with the non-ticking TE handler that repolainen implemented, the AE disks became even less attractive

**Comment by Prometheus0000** [comment]
It's still more expensive, as we have the intermediary item processor item. And the whole idea is to get away from recursive crafting.

**Comment by Prometheus0000** [comment]
We also have the very expensive circuits in ours, whereas vanilla does not.

**Comment by GTNH-Afx237v7** [comment]
then i suggest we take the discussion back to the RFC thread; I'll post what botn365 mentioned in the discord so we have a different perspective as well

**Comment by Prometheus0000** [comment]
Why though? Just make the current tier chip more expensive?

**Comment by Prometheus0000** [comment]
Also, what do you mean by 'chip', there's 3 different things it could be?

**Comment by Dream-Master** [comment]
Why more expensive? if you se only one instead of 4 from previous tier? Or we can change the recipes of the ae ships. this recipes are very old and outdated too.

**Comment by Prometheus0000** [comment]
What? You're the one that wanted to make the recipes more expensive? I'm saying make the item processor more expensive instead of using the previous tier of item processor. Which I already said on discord. Did you read the discussion?

**Comment by Dream-Master** [comment]
You made the recipe very much more cheaper and i want to take care make the recipe more balanced. With the changes i will probably doing we not 99% cheaper but maybe 80%.

**Comment by Prometheus0000** [comment]
And I'm asking why you want to add 
image
A previous tier of these in, when you can simply change their recipe?
image
Like add in some circuits or something.

Personally, I till think they're expensive enough, but I dunno.

**Comment by Dream-Master** [comment]
This recipe is very old. We could add a solder and a control circuit. Or /and add some fine wire or some things make the recipe looks a bit nicer.I dont speak about changes you did to components i speak about minimal changes and adjustments :)

File: scripts/Extra-Cells.zs [hunk 1]
```diff
@@ -619,16 +619,16 @@ Assembler.addRecipe(<extracells:craftingstorage:2>, <appliedenergistics2:tile.Bl
 Assembler.addRecipe(<extracells:craftingstorage:3>, <appliedenergistics2:tile.BlockCraftingUnit>, <extracells:storage.component:3>, 400, 1920);
 
 // --- Storage Cell Component - 256K
-Assembler.addRecipe(StorageComponent256K, [DataCircuit * 4, TitaniumPlate * 4, <dreamcraft:item.EngineeringProcessorItemEmeraldCore>, <appliedenergistics2:item.ItemMultiMaterial:8> * 4], null, 200, 1920);
+Assembler.addRecipe(StorageComponent256K, [DataCircuit * 4, TitaniumPlate * 4, <dreamcraft:item.EngineeringProcessorItemEmeraldCore>, <appliedenergistics2:item.ItemMultiMaterial:8> * 4, <gregtech:gt.integrated_circuit:9> * 0], null, 200, 1980);
```
Comment (Dream-Master) [inline scripts/Extra-Cells.zs:623]: 1920 was ok

**Review by Dream-Master — CHANGES_REQUESTED** [review]

**Comment by Prometheus0000** [comment]
Forgot to update, they now look like https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7839#issuecomment-826104395

**Comment by Dream-Master** [comment]
lets use a 3:1 ratio not ix it up. And yes plates are to cheap

**Pull request closed without merging**